code/controllers/subsystem/non_firing/SSassets.dm

```diff
@@ -2,15 +2,37 @@ SUBSYSTEM_DEF(assets)
 	name = "Assets"
 	init_order = INIT_ORDER_ASSETS
 	flags = SS_NO_FIRE
-	var/list/cache = list()
+	var/list/datum/asset_cache_item/cache = list()
```

Comment:
you can't restrict lists to a type like this, just document this variable and specify which types it contains in the comment